src/main/webapp/app/entities/flair-bi/filter/filter-penal-pin.component.html

```diff
@@ -0,0 +1,13 @@
+<form class="form-inline pin-filter" >
+    <div class="form-group" ng-repeat="dimension in vm.dimensions">
+        <div class="filter-section">
+            <label class="dimension-label">{{dimension.name}}</label>
+            <div class="dimensionFilter" ng-dropdown-multiselect="" options="vm.list[dimension.name]" selected-model="vm.selectedFilter[dimension.name]"
+                checkboxes="true" selectedToTop="true" extra-settings="vm.setting1"></div>
```

Comment:
this is not valid name vm.setting1...can u rename to something suitable to functionality